src/cli/README_BR.md

```diff
@@ -206,3 +208,37 @@ Clear a previously set preference value for advertising Route Info Options (e.g.
 > br rioprf clear
 Done
 ```
+
+### routeprf
+
+Usage: `br routeprf`
+
+Get the preference used for publishing routes in Thread Network Data.
```

Comment:
```suggestion
Get the preference used for publishing routes in Thread Network Data. This may be the automatically determined route preference, or an administratively set fixed route preference - if applicable.
```

To clarify what is exactly returned here.